docs/antimony.rst

```diff
@@ -1393,4 +1425,4 @@ Further Reading
 ---------------
 
 * Lucian Smith's `example models <http://antimony.sourceforge.net/antimony-examples.html>`_ show how to use the `comp package <http://sbml.org/Documents/Specifications/SBML_Level_3/Packages/comp>`_.
-* `Antimony's manual <http://antimony.sourceforge.net/Tutorial.pdf>`_ in PDF format.
\ No newline at end of file
+* `Antimony's manual <http://antimony.sourceforge.net/Tutorial.pdf>`_ in PDF format.
```

Comment:
the "added" line is here.

Reply:
Oh, ha.  OK, then!  I'm going to guess 'trailing space change', then.